vars.go

```diff
@@ -33,6 +33,14 @@ const (
 	AttributeTimeBasedAuthenticatedWriteAccess VariableAttributes = 1 << 5
 	AttributeAppendWrite                       VariableAttributes = 1 << 6
 	AttributeEnhancedAuthenticatedAccess       VariableAttributes = 1 << 7
+
+	// unix.EFIVARFS_MAGIC is 0xde5e81e4, which is outside of
+	// `long` range on 32bit platforms. And kernel does not care,
+	// and overflows it to -28, but golang fails to compile the
+	// code as it strictly checks the const bounds meaning
+	// unix.EFIVARFS_MAGIC is int64. Define a second
+	// EFIVARFS_MAGIC2 here.
+	EFIVARFS_MAGIC2 int64 = -28
```

Comment:
actually i'm not too sure this is correct. need to check this on all the arches.